buildspec.yml

```diff
@@ -930,11 +940,11 @@ phases:
       - aws ecr-public get-login-password --region $AWS_DEFAULT_REGION | docker login --username AWS --password-stdin public.ecr.aws/$PUBLIC_ECR_ALIAS
   build:
     commands:
-      - echo Building the Docker image...
+      - echo Building the Docker image...          
       - cd $IMAGE_DIR/$IMAGE_TAG
       - docker build -t $IMAGE_REPO_NAME:$IMAGE_TAG .
       - docker tag $IMAGE_REPO_NAME:$IMAGE_TAG public.ecr.aws/$PUBLIC_ECR_ALIAS/$IMAGE_REPO_NAME:$IMAGE_TAG
   post_build:
     commands:
       - echo Pushing the Docker image...
-      - docker push public.ecr.aws/$PUBLIC_ECR_ALIAS/$IMAGE_REPO_NAME:$IMAGE_TAG
+      - docker push public.ecr.aws/$PUBLIC_ECR_ALIAS/$IMAGE_REPO_NAME:$IMAGE_TAG
```

Comment:
formatter?

Reply:
https://github.com/cypress-io/cypress-docker-images/commit/8185fe350f1b651683b3a1ca2d7261e37152c86a#diff-69387ac97f1b775f19989fc28150f89e785406cbff04643a969f5e396573dd5c removed the whitespace, but `./generate-included-config.js` adds it back